.github/workflows/test-deploy-staging.yaml

```diff
@@ -188,10 +194,11 @@ jobs:
             echo "Failed getting RDS host with name $CLUSTER_NAME"
             exit 1
           fi
+          PG_PORT=5432
```

Comment:
there is no point parametrising `PG_PORT` for the time being -- it is used only once